src/app/components/InProgressItem/index.tsx

```diff
@@ -53,6 +71,17 @@ export const InProgressItem = (props: Props) => {
       >
         {t("button.continue")}
       </ButtonCustomization>
+      <ModalCustomization
+        open={openPopup}
+        handleCancel={handleClosePopup}
+        handleAgree={handleSubmitContinue}
+        actionDefault
+        title="Bạn có chắc chắn muốn làm tiếp bài thi không?"
+        textAgreeBtn="Continue"
+        colorBtn="success"
+      >
+        <div></div>
```

Comment:
Why have a empty div inside ModalCustomizaion?

Reply:
prop children in ModalCustomization component, to show contentText